orderapi/src/main/java/fi/hel/verkkokauppa/order/logic/RecurringOrderValidationLogic.java

```diff
@@ -32,7 +33,7 @@ public void validateSupportedPeriodFrequency(RecurringOrderDto dto, Errors error
 	public void validateStartDate(RecurringOrderDto dto, Errors errors) {
 		final LocalDate now = LocalDate.now();
```

Comment:
Just double-checking whether you meant to compare at date level or at time level. Now at date level means 2021-01-01 00:00:00

Reply:
I thinks this function should be refactored later because we use LocalDateTime now., so this is good enouh for now.